test/powershell/engine/Module/NewModuleManifest.Tests.ps1

```diff
@@ -9,14 +9,7 @@ Describe "New-ModuleManifest tests" -tags "CI" {
     }
 
     BeforeAll {
-        if ($IsWindows)
-        {
-            $ExpectedManifestBytes = @(255,254,35,0,13,0,10,0)
-        }
-        else
-        {
-            $ExpectedManifestBytes = @(35,10)
-        }
+        $ExpectedManifestBytes = @(35,10)
```

Comment:
In BeforeEach, `New-Item` should pipe to $null. GitHub won't let me comment on that line since it is not near the change.

Reply:
Will add